boot/term.sh

```diff
@@ -31,20 +31,18 @@ tar -xvzf client.tar.gz -C .;
 if [ -f client/start.sh ]; then
   exec bash client/start.sh
 else
-  cd client/node_modules/usb/;
-  npm install --build-from-source --verbose;
-
-  cd ../t2-cli/node_modules/usb;
-  npm install --build-from-source --verbose;
-
   # cd into the directory that we store the Node tarball
-  cd ~/client/node/;
+  cd ~/client/client/node/;
```

Comment:
Yeah, the file structure is a little confusing. We basically have:
client
-- config.json
-- some other files
-- client
----index.js
----...